demos/OptimalPlanning.cpp

```diff
@@ -167,6 +169,11 @@ ob::PlannerPtr allocatePlanner(ob::SpaceInformationPtr si, optimalPlanner planne
             return std::make_shared<og::InformedRRTstar>(si);
             break;
         }
+        case PLANNER_RRTSTARSMART:
```

Comment:
I appreciate that these are in the same order as the enum definition. If you reorder that, please keep this the same.